cryosparc/command.py

```diff
@@ -100,24 +111,35 @@ def __init__(
     def _get_callable(self, key):
         def func(*args, **kwargs):
             params = kwargs if len(kwargs) else args
-            data = {
-                "jsonrpc": "2.0",
-                "method": key,
-                "params": params,
-                "id": str(uuid.uuid4()),
-            }
+            data = {"jsonrpc": "2.0", "method": key, "params": params, "id": str(uuid.uuid4())}
             res = None
             try:
                 with make_json_request(self, "/api", data=data, _stacklevel=4) as request:
                     res = json.loads(request.read())
-            except CommandClient.Error as err:
-                raise CommandClient.Error(
-                    self, f'Did not receive a JSON response from method "{key}" with params {params}', url=self._url
+            except CommandError as err:
+                raise CommandError(
```

Comment:
No, I've updated the `make_request`/`make_json_request` implementations so that they only ever raise `CommandError`. That way we can capture the error code and data, and re-raise it here with the JSONRPC key and params information in the error message.